sdk/synapse/synapse-artifacts/src/models/parameters.ts

```diff
@@ -6,7 +6,7 @@
  * Changes may cause incorrect behavior and will be lost if the code is regenerated.
  */
 
-import type {
+import {
```

Comment:
why there is no api-version change? is it because these features are added in existing version?